udine1/index.html

```diff
@@ -10,9 +10,146 @@
     <link rel="icon" type="image/png" href="favicon-16x16.png" sizes="16x16" />
     <link rel='stylesheet prefetch' href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css'>
     <link rel="stylesheet" href="css/style.css">
+	<link rel="stylesheet" href="assets/font-awesome/font-awesome/css/font-awesome.min.css" type="text/css">
   </head>
 
   <body>
+  
+	<!-- Parallax
+      ================================================== -->
+	  
+		<div class="container-fluid">
+			<div class="row parallax">
+				<div class="col-lg-4 col-md-4 col-xs-12">
+					<section class="help">
+					<img src="img/icon_1_parallax.png">
+					<h3>Hai bisogno di aiuto?</h3>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam lectus risus</p>
+					<button class="btn light-blue">Centro assistenza</button>
+					</section>
+				</div>
+				<div class="col-lg-4 col-md-4 col-xs-12">
+					<section class="riservata">
+					<img src="img/icon_2_parallax.png">
+					<h3>La tua area riservata</h3>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam lectus risus</p>
+					<button class="btn light-blue">Area riservata</button>
+					</section>
+				</div>
+				<div class="col-lg-4 col-md-4 col-xs-12">
```

Comment:
`class="col-lg-4 col-md-4 col-xs-12"` is redundant and can be simplified to `class="col-xs-12 col-md-4"`